templates/home/_base.scss

```diff
@@ -50,6 +50,26 @@
 		margin: 1em 0px;
 	}
 
+	// All services button
+	.all-services-btn {
```

Comment:
FYI - 
See CSS used for the super task button here: https://github.com/wet-boew/GCWeb/blob/master/sites/baseline/_buttons.scss#L16
And here a working example/code sample: https://github.com/wet-boew/GCWeb/blob/master/templates/content-page/content-en.md?plain=1#L24

Reply:
I removed the custom css and used the call-to-action button class for styling.